pavelib/servers.py

```diff
@@ -48,40 +54,63 @@ def run_server(system, settings=None, port=None, skip_assets=False, contracts=Fa
 @needs('pavelib.prereqs.install_prereqs')
 @cmdopts([
     ("settings=", "s", "Django settings"),
+    ("asset-settings=", "a", "Settings file used for updating assets. Defaults to settings if not provided."),
```

Comment:
The default behavior is that the asset_settings just follows the regular settings. It is only for the optimized case that the user wants to have a different settings file for asset generation than for running the server.